.gitignore

```diff
@@ -54,6 +54,7 @@ wp-content/wp-cache-config.php
 wp-content/tests/
 wp-content/logs/
 wp-content/mu-plugins/mu.local.php
+wp-content/mu-plugins/local-by-flywheel-live-link-helper.php
```

Comment:
This is auto-generated when using Local by Flywheel for a local environment and Live Links is enabled. I've used this to speed up testing across devices IRL and w/ Browserstack.

Reply:
Really minor, but this probably is better adde under the `# IDE generated files #` section above.

Reply:
Good call, moved it there.